app/models/appointment_slot.rb

```diff
@@ -1,8 +1,14 @@
 class AppointmentSlot < ActiveRecord::Base
+  include PushUpdatesMinimal
+
   validates :quota, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
   validate :no_duplicate, on: [:create, :update]
   before_save :clean_timestamp
 
+  after_save :push_changes
+  after_destroy :push_changes
+  push_targets [ Channel::STOCK_CHANNEL ]
```

Comment:
Do these also appear on Browse app?

Reply:
also, I wonder if this could be ORDER_FULFILMENT_CHANNEL as it is only related to the order part of the stock app.

Reply:
They do appear on browse, but they are fetched every time we open the schedule page, so it's not a persisted resource in the app.

Reply:
for reference: Will keep `STOCK_CHANNEL`  after a chat, as non fulfilment roles with `can_manage_settings` are allowed to modify slots